omnisharp.el

```diff
@@ -350,7 +350,7 @@ locations in the json."
 cursor at that location"
   (let* ((element-line (cdr (assoc 'Line element)))
          (element-column (cdr (assoc 'Column element)))
-         (element-filename (cdr (assoc 'Filename element)))
```

Comment:
Looks like this has been a mistake. I'll change this to be in line with the others ("FileName").